packages/lib/src/toolbar/controls/Selector/Selector.tsx

```diff
@@ -87,7 +80,7 @@ function Selector<T extends string>(props: Props<T>) {
 
   function handleSelect(index: number, option: T) {
     setSelectedIndex(index);
-    toggle(false);
+    context.onOpenChange(false);
```

Comment:
In case I decide to hide the `isOpen` state and `toggle` setter (along with `useFloating`) inside a reusable custom hook.